services/uploads/serverless.yml

```diff
@@ -177,6 +218,102 @@ resources:
             - ServerSideEncryptionByDefault:
                 SSEAlgorithm: AES256
         AccessControl: Private
+    BucketAVScanRole:
```

Comment:
This role shouldn't be added as it's not used anywhere.  It's a relic of how we used to do roles.  
See https://github.com/CMSgov/macpro-quickstart-serverless/pull/266 (linked elsewhere too)... moved us to the serverless native lambda role management.